### PR TITLE
Add framework of CI testing for windows CWS

### PR DESCRIPTION
**Pull request by derekwbrown**
### What does this PR do?

Does NOT actually add any meaningful tests yet.

This PR is intended simply to get the framework in place for adding meaningful tests, so that multiple groups can work in parallel.
### Motivation

Write more tests.
### Describe how to test/QA your changes

Run a pipeline.  Does the new test job run

### Reviewer's Checklist

- [x] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [x] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [x] Changed code has automated tests for its functionality.
- [x] Adequate QA/testing plan information is provided. Except if the `qa/skip-qa` label, with required either `qa/done` or `qa/no-code-change` labels, are applied.
- [x] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: .gitlab/source_test/ebpf.yml [hunk 1]
```diff
@@ -53,6 +53,33 @@
     paths:
       - $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files
 
+.tests_windows_secagent:
```
Comment (brycekahle) [inline .gitlab/source_test/ebpf.yml:56]: Can you add these to a different file in this directory? They have nothing to do with eBPF.

Comment (brycekahle) [reply .gitlab/source_test/ebpf.yml:56]: I realize `tests_windows_sysprobe` is already in this file, but that should probably be moved too.

**Review by ISauve — APPROVED** [review]

File: tasks/winbuildscripts/secagent.ps1 [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    $archflag = "x86"
+}
+& .\tasks\winbuildscripts\pre-go-build.ps1 -Architecture "$archflag" -PythonRuntimes "$Env:PY_RUNTIMES"
+
+#& inv -e lint-go --build system-probe-unit-tests --targets .\pkg
+#$err = $LASTEXITCODE
+#if ($err -ne 0) {
+#    Write-Host -ForegroundColor Red "lint-go failed $err"
+#    [Environment]::Exit($err)
+#}
```
Comment (clarkb7) [inline tasks/winbuildscripts/secagent.ps1:33]:
```suggestion
```

**Review by clarkb7 — APPROVED** [review]

**Review by paulcacheux — APPROVED** [review]

**Review by safchain — APPROVED** [review]

**Review by spencergilbert — APPROVED** [review]

**Pull request merged**